templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+    <h1>{{ object|yesno:"Update, Create" }} a Manufacturer</h1>
+  
+  {% if error %}
+    <p>{{ error }}</p>
+  {% endif %} 
+
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <br>
+    <input class="btn btn-primary" type="submit" value="Submit">
+  </form>
```

Comment:
According to the checklist, you should use 2 whitespaces indentations in your `.html` files. Please ensure that you are following this style guide.